content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -57,8 +57,11 @@ br_netfilter
 EOF
 
 cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+
 net.bridge.bridge-nf-call-ip6tables = 1
 net.bridge.bridge-nf-call-iptables = 1
+net.ipv4.ip_forward = 1
+net.ipv6.ip_forward = 1
```

Comment:
we already mention `net.ipv4.ip_forward = 1` in https://kubernetes.io/docs/setup/production-environment/container-runtimes/

but i guess it can be added here explicitly, but not ipv6.